in_toto/runlib.py

```diff
@@ -319,7 +319,7 @@ def record_artifacts_as_dict(artifacts, exclude_patterns=None,
 
   return artifacts_dict
 
-def execute_link(link_cmd_args, record_streams):
+def execute_link(link_cmd_args, record_streams, quiet):
```

Comment:
Is there a default for `quiet`?

Reply:
```suggestion
def execute_link(link_cmd_args, record_streams, quiet=True):
```

going by the call below

Reply:
Oh, yes! I should've included this! Thanks for catching that